### PR TITLE
Update CSES Road Construction.mdx

### PR DESCRIPTION
**Pull request by TruaShamu**
Added java sol to cses1676 road construction

### CODE REVIEW

**Review by dongliu0426 — APPROVED** [review]

**Comment by thecodingwizard** [comment]
oh crap I merged the wrong PR. @HarryWangATX @Farrius please still review this if you can :P

**Comment by HarryWangATX** [comment]
Ill revert it for now.

**Comment by HarryWangATX** [comment]
wait, nvm

File: solutions/CSES Road Construction.mdx [hunk 1]
```diff
@@ -2,8 +2,12 @@
 id: cses-1676
 source: CSES
 title: Road Construction
-author: Oscar Garries
+author: Oscar Garries, Sofia Yang
 ---
+<LanguageSection>
```
Comment (HarryWangATX) [inline solutions/CSES Road Construction.mdx:7]: add a implementatino heading here.

Comment (HarryWangATX) [inline solutions/CSES Road Construction.mdx:7]: also add time complexity

File: solutions/CSES Road Construction.mdx [hunk 2]
```diff
@@ -45,3 +49,87 @@ int main () {
...
 }
 ```
+</CPPSection>
+
+<JavaSection>
+
+```java
+
+
+import java.io.BufferedReader;
```
Comment (HarryWangATX) [inline solutions/CSES Road Construction.mdx:59]:
```suggestion
import java.io.BufferedReader;
```

**Comment by TruaShamu** [comment]
resolved :)

**Comment by HarryWangATX** [comment]
i don't see ur commmits, try recommiting or new PR

**Pull request merged**